src/init.cpp

```diff
@@ -954,10 +954,14 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     if (mapArgs.count("-minrelaytxfee"))
     {
         CAmount n = 0;
-        if (ParseMoney(mapArgs["-minrelaytxfee"], n) && n > 0)
-            ::minRelayTxFee = CFeeRate(n);
-        else
+        if (!ParseMoney(mapArgs["-minrelaytxfee"], n))
             return InitError(AmountErrMsg("minrelaytxfee", mapArgs["-minrelaytxfee"]));
+#ifdef ENABLE_WALLET
+        if (!fDisableWallet && n > HIGH_TX_FEE_PER_KB)
+            InitWarning(AmountHighWarn("-minrelaytxfee") + " " +
+                        _("This is the minimum transaction fee you pay on every transaction."));
```

Comment:
Minrelayfee influences more than the fee you pay yourself. Isn't it a bit confusing to advertize it as just that?